categoricalHmap/script.js

```diff
@@ -0,0 +1,10 @@
+// Sandbox for categoricalHmap
+  var data = [
+    {
+      z: [[1, 20, 30, 50, 1], [20, 1, 60, 80, 30], [30, 60, 1, -10, 20]],
+      x: ['Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday'],
+      y: ['Morning', 'Afternoon', 'Evening'],
+      type: 'heatmap'
+    }
+  ];
+  Plotly.newPlot('myDiv', data);
```

Comment:
This directory keeps sneaking into the root. Can't approve this PR until it's in a sandbox.